### ISSUE TITLE
Font without antialiasing

### ISSUE THREAD

**Issue by freeconst** [wsdfhjxc/do-it-yourself-bar#3]
Kubuntu 20.04
Plasmashell 5.18.5
DIY-bar compiled from current source by applied script.

Font antialiasing is another (more blured) to other applets and other system components. Can see it on screenshot below. Also "fives" on RAM block is another then in clock one (same font, same kerning).
![Screenshot_20210120_121650](https://user-images.githubusercontent.com/5403268/105159315-0079de80-5b20-11eb-9ee4-592ca6ddefe9.png)

**Comment by wsdfhjxc** [comment]
The plasmoid uses QML's Text components, instead of Label components, and the latter are widely used in most Plasma applets. That's because font rendering in Plasma has been a trainwreck, mostly in regard to kerning. For example, while using a Label component, Roboto font at 12px size looks terrible in comparison to GTK/Pango, which has excellent font rendering.

The Text component is better in that aspect, but there are still other issues with fonts at various sizes. I very much prefer it to be a little bit blurry, but have proper kerning and letter proportions, so it looks like in GTK/Pango. However, it's not that there's no antialiasing at all (as the title suggests). It's just that Text components are rendered differently than Label components.

I haven't tried the latest Plasma/Qt versions, so I don't know if the QML font issues have been addressed yet, but if I remember correctly, there's been work in progress, and improvements scheduled for Qt 5.15. Regardless, I'll experiment with QML's Label component once again, and see if the kerning/sizing isn't as horrible as it used to be the last time I tried it.

**Comment by freeconst** [comment]
Yes, your truth! Antialiasing is present, but different from the rest of system. I also had an idea to test on a new plasma, but now there is not enough time. However, I tested other applets and came to following conclusion: Kargos is close to system rendering, but Command Output (I don't remember exact name) has exactly same font blur effect. But naturally, the functionality of both is weaker.
